k8s/demo/common/money-claims/v1.yaml

```diff
@@ -0,0 +1,188 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: demo-v1
+  namespace: money-claims
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: demo-v1
+  chart:
+    repository: https://hmcts.azurecr.io/helm/v1/repo/
+    name: chart-cmc
+    version: 0.0.1
+  values:
+    ### CMC SERVICES ###
+    cmc-claim-store:
+      java:
+        releaseNameOverride: demo-v1-cmc-claimstore
+        secrets: 
+          GOV_NOTIFY_API_KEY:
+              secretRef: cmc-demo-notify-api-key
+              key: key
+          OAUTH2_CLIENT_SECRET:
+              secretRef: cmc-demo-oauth-client-secret
+              key: key
+          IDAM_CASEWORKER_ANONYMOUS_USERNAME:
+              secretRef: cmc-demo-anonymous-caseworker-username
+              key: key
+          IDAM_CASEWORKER_ANONYMOUS_PASSWORD:
+              secretRef: cmc-demo-anonymous-caseworker-password
+              key: key
+          IDAM_CASEWORKER_SYSTEM_USERNAME:
+              secretRef: cmc-demo-system-update-username
+              key: key
+          IDAM_CASEWORKER_SYSTEM_PASSWORD:
+              secretRef: cmc-demo-system-update-password
+              key: key
+        environment:
+          SPRING_MAIL_HOST: mta.reform.hmcts.net 
+          SPRING_MAIL_PORT: 25
+          CLAIM_STORE_DB_HOST: demo-v1-postgres
+          CMC_DB_HOST: demo-v1-postgres
+          CORE_CASE_DATA_API_URL: http://demo-v1-data-store-api
+          FEATURE_TOGGLES_CCD_ENABLED: true
+          FEATURE_TOGGLES_CCD_ASYNC_ENABLED: false
+          FEATURE_TOGGLES_SAVE_CLAIM_STATE_ENABLED: true
+          FEATURE_TOGGLES_ASYNC_EVENT_OPERATIONS_ENABLED: true
+          FEATURE_TOGGLES_EMAILTOSTAFF: true
+          IDAM_API_URL: https://idam-api.demo.platform.hmcts.net
```

Comment:
it would be good to group together the per demo specific config, and the static demo config. i.e. move this up and any other urls down?